src/climate_learn/metrics/utils.py

```diff
@@ -0,0 +1,25 @@
+# Standard library
+from dataclasses import dataclass
+from typing import List
+
+# Local application
+import numpy.typing as npt
+import torch
+
+
+@dataclass
+class MetricsMetaInfo:
```

Comment:
It's true that not all metrics use the metainfo. E.g., vanilla MSE does not use any of these. However, I chose to make this class because (1) I think it's reasonable to assume that these variables should be available for any climate dataset, (2) it allowed me to build a unified initialization function signature for the metrics classes.